bellman/src/lib.rs

```diff
@@ -26,6 +26,10 @@ mod multiexp;
 #[cfg(test)]
 mod tests;
 
+pub fn get_chunk_size(total: usize) -> usize {
```

Comment:
Please add a method doc.

Reply:
Added in [new commit](https://github.com/MystenLabs/phase2-bn254/pull/1/commits/af516018062a5c87eacb44c572c8f80c858155a8#diff-712193d9f325ef870c241172009a26cfe5bed8ace06fc0c81cf660e9bbe654bcR29).